packages/studioCMS/src/integrations/studioCMSDashboard/routes/dashboard/pages/edit/page-list.astro

```diff
@@ -0,0 +1,121 @@
+
+<Layout 
+    sideBarActiveItemID="edit-pages" 
+    title={pageTitle}
+    description={contextConfig.description} >
+
+    <div class="flex grow w-full pb-4">
+        <sl-breadcrumb>
+            <span slot="separator">/</span>
+            <sl-breadcrumb-item href={dashboardIndex}>Dashboard
+                <sl-icon slot="prefix" name="columns-gap"></sl-icon>
+            </sl-breadcrumb-item>
+            <sl-breadcrumb-item href={pageEdit}>Existing Pages</sl-breadcrumb-item>
+        </sl-breadcrumb>
+    </div>
+
+    <div class="container">
+
+        <div class="text-5xl font-bold py-8">Existing Pages</div>
+
+        <div class="py-2">
+            <div class="relative">
+                <table class="w-full text-left rtl:text-right">
+                    <thead class="uppercase bg-base-300 text-inherit">
+                        <tr>
+                            <th scope="col" class="px-6 py-3">Page Title</th>
+                            <th scope="col" class="px-6 py-3">Package Relation</th>
+                            <th scope="col" class="px-6 py-3">Publish Date</th>
+                            <th scope="col" class="px-6 py-3">Edit</th>
+                            <th scope="col" class="px-6 py-3">Delete</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        { 
+                            pages.map(async (page) => {
+                                return (
+                                    <tr class="bg-base-200 text-inherit">
+                                        <th scope="row" class="px-6 py-4 font-medium whitespace-nowrap">
+                                            {page.title}
+                                        </th>
+                                        <td class="px-6 py-4">
+                                            {fixPackageLabel(page.package)}
+                                        </td>
+                                        <td class="px-6 py-4">
+                                            <FormattedDate date={page.publishedAt}/>
+                                        </td>
+                                        <td class="px-6 py-4">
+                                            <sl-button variant="success" href={await getEditRoute(page.slug)} size="small" type="primary" pill outline>
+                                                Edit
+                                            </sl-button>
+                                        </td>
+                                        <td class="px-6 py-4">
+                                            { permissionLevel === 'admin' ? <sl-button variant="danger" href={await getDeleteRoute(page.slug)} size="small" type="primary" pill outline>
+                                                Delete
+                                            </sl-button> : <sl-button variant="danger" href={await getDeleteRoute(page.slug)} size="small" type="primary" pill outline disabled>
```

Comment:
It's best to just not show the button if they don't have permission

Reply:
Removing the button makes the table look funny :sweat_smile:

Reply:
Hmmm could just wrap the buttons in a flex row with a min width or something like that?

Reply:
done as of [b8f4d1a](https://github.com/astrolicious/studiocms/pull/77/commits/b8f4d1a47f4013654d869cb3a81c39f572e96cd1)